pkg/ccl/streamingccl/streamclient/client.go

```diff
@@ -90,7 +90,9 @@ type Client interface {
 	// Complete completes a replication stream consumption.
 	Complete(ctx context.Context, streamID streampb.StreamID, successfulIngestion bool) error
 
-	PriorReplicationDetails(ctx context.Context, tenant roachpb.TenantName) (string, hlc.Timestamp, error)
+	PriorReplicationDetails(
```

Comment:
I decided "sourceID" and "srcID" are confusing here, since "source" or "src" also is used in contrast to "destination" or "dst" elsewhere in code that handles these IDs. I've revised to to call it a "historyID", regardless of whether it is being used on the src or dst, to reduce the cognitive load a bit. WDYT?

Reply:
yeah this explanation works for me.